test/duplicates-v3.e2e-spec.ts

```diff
@@ -167,12 +167,12 @@ describe('Deposits in case of duplicates', () => {
   test(
     'skip deposits for module if find duplicated key across operator',
     async () => {
-      const currentBlock = await providerService.provider.getBlock('latest');
-
-      // TODO: mine new block instead
+      // TODO: mine new block instead (? @Anna)
```

Comment:
@Amuhar you don't remember what that means?

Reply:
to mine a new block we make deposit in this test 
instead we can use await providerService.provider.send('evm_mine', []);
please ignore this todo now, in my new pull request i resolved this todo